tools/cldr-apps/src/main/java/org/unicode/cldr/web/api/Auth.java

```diff
@@ -82,10 +83,17 @@ public Response login(
             String userIP = WebContext.userIP(hreq);
             CookieSession session = null;
             if (!request.isEmpty()) {
-                UserRegistry.User user =
-                        CookieSession.sm.reg.get(request.password, request.email, userIP);
+                UserRegistry.User user;
+                try {
+                    user = CookieSession.sm.reg.get(request.password, request.email, userIP);
+                } catch (LogoutException e) {
+                    user = null;
+                }
+                if (user == null) {
+                    user = AuthSurveyDriver.createTestUser(request.password, request.email);
+                }
```

Comment:
this is a little awkward since UserRegistry.get sometimes throws LogoutException and sometimes just returns null without throwing an exception -- currently it throws LogoutException for a new webdriver user, but I didn't want to rely on that here, so this also handles the null return possibility